src/main/java/io/antmedia/AppSettings.java

```diff
@@ -732,6 +732,10 @@ public class AppSettings implements Serializable{
 	 */
 	private static final String SETTINGS_CLUSTER_COMMUNICATION_KEY = "settings.clusterCommunicationKey";
 
+	private static final String SETTINGS_WEBHOOK_RETRY_COUNT = "settings.webHookRetryCount";
```

Comment:
No need these static fields any more. Just only use the exact field name

Reply:
done

Reply:
Thank you